### ISSUE TITLE
Create a doc on how to edit for ourselves and deploy it to vercel

### ISSUE THREAD

**Issue by tassosmak** [NuroDev/nuro.dev#62]
I would appreciate a tutorial on how to edit this page for ourselves and how to deploy it to Vercel

**Comment by HelloMikko** [comment]
No need to create a doc, its all litearlly in the src/data folder. look at the codes carefully. `ctrl+f` and find `nuro.dev` or `nurodev`. however that is all I will say. But last thing, before you decide to want to use someone elses open-source, you should first know what and how before. I say this with no offense, so dont assume, I myself found this in the matter of seconds.

**Comment by tassosmak** [comment]
Yes, you are right however what I failed to mention is that my main point was how I could deploy this page to Vercel.

**Comment by codemikko** [comment]
> Yes, you are right however what I failed to mention is that my main point was how I could deploy this page to Vercel.

Well netlify and vercel works the exact same way. Create an account( I would advise you to use the github login because its easier down the road) > and adding a project would be click > Add New/Project. From there your github repo to where you have what ever project you want like the nuro.dev project for an example, choose that repo and click deploy, done. Enjoy. Remember this also. If the original Developer or any application or project does not leave a How-To tutorial, its because he or she really dont want to share their own work. Is it illegal to still use it? Not really per-say. however, giving them the respect, should always be something that is what it is. However, if they dont say anything, then... well, thats because youll have to figure out things yourself if you want to just copy their work.
